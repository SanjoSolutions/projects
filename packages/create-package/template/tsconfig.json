{
  "extends": "../../tsconfig-base.json",
  "compilerOptions": {
    "rootDir": "./src/",
    "outDir": "./built/",
    "module": "CommonJS",
    "lib": ["ES2020"]
  },
  "references": []
}
