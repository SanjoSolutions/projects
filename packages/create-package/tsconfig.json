{
  "extends": "../../tsconfig-base.json",
  "compilerOptions": {
    "rootDir": "./src/",
    "outDir": "./",
    "lib": ["ESNext"],
    "tsBuildInfoFile": "tsconfig.tsbuildinfo"
  },
  "include": ["./src/**/*"],
  "references": [
    {
      "path": "../diff-folders"
    },
    {
      "path": "../escape-for-reg-exp"
    },
    {
      "path": "../exec"
    },
    {
      "path": "../make-temporary-directory"
    },
    {
      "path": "../read-file"
    },
    {
      "path": "../test-create-package"
    },
    {
      "path": "../write-file"
    }
  ]
}
