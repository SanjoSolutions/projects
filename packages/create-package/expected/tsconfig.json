{
  "extends": "../../tsconfig-base.json",
  "compilerOptions": {
    "rootDir": "./src/",
    "outDir": "./built/",
    "lib": ["ESNext"],
    "tsBuildInfoFile": "tsconfig.tsbuildinfo"
  },
  "include": ["./src/**/*"],
  "references": []
}
